calico-cloud_versioned_docs/version-3.17/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -1,73 +1,75 @@
 ---
-description: Scan images using CLI scanner.
+description: Scan images in your build pipeline using Image Assurance.
 ---
 
-# Scan images using the CLI scanner
+# Integrate Image Assurance scanner in your build pipeline
 
 ## Big picture
 
-Scan images manually or via CI/CD pipeline for vulnerabilities using {{prodname}} Image Assurance CLI scanner.
+Integrate the Image Assurance scanner into your build pipeline and private registries to ensure builds are checked by Image Assurance before deployment. You can also use the scanner offline and on-demand for ad hoc scanning and emergency patching. For a real-world use case, see [Hands-on guide: How to scan and block container images to mitigate SBOM attacks](https://www.tigera.io/blog/hands-on-guide-how-to-scan-and-block-container-images-to-mitigate-sbom-attacks/).
+
+The Image Assurance scanner is available in a containerized version so you can easily update it as part of a container-or Kubernetes-based pipeline.
```

Comment:
@bmckercher123 I think we should actually mention how the user can download the containerized CLI. 

I originally had thought that we might want to present the containerized CLI as a separate page, at the same level of this page and the cluster scanner page. But perhaps it's better suited as a section on this page, because it's technically still the CLI scanner. 

cc @Brian-McM

Reply:
Last discussion with Brian, we decided to put this information in a blog and point to it in this doc as an example.

Reply:
I think we were going to put the details about how to integrate with a specific CI / CD pipeline in a blog post, but we were  going to still mention that we have a containerised version in the docs.

Something like "We have a containerised version of the cli that can be integrated and can be downloaded here <include link>. For details on how to integrate with a specific pipeline, look here <link to harness blog>".

Reply:
It is already in the doc here:
Integrate the Image Assurance scanner into your build pipeline and private registries to ensure builds are checked by Image Assurance before deployment. You can also use the scanner offline and on-demand for ad hoc scanning and emergency patching. For a real-world use case, see [Hands-on guide: How to scan and block container images to mitigate SBOM attacks](https://www.tigera.io/blog/hands-on-guide-how-to-scan-and-block-container-images-to-mitigate-sbom-attacks/).
**The Image Assurance scanner is available in a containerized version so you can easily update it as part of a container-or Kubernetes-based pipeline.**

Reply:
@bmckercher123 I think we need to have instructions for here on how the user can pull the containerized CLI scanner (similar to how we have instructions for how to download the binary version of the CLI scanner).

Otherwise, how is a customer supposed to self-serve and access the containerized version?

Reply:
Sorry I just realized it would be helpful if I gave you the details on the location of the containerized CLI scanner. 

This is the Quay repo for it: 
https://quay.io/repository/tigera/image-assurance-scanner-cli

The exact image should be: 
```
quay.io/tigera/image-assurance-scanner-cli:v1.8.4
```